applications/ipc_radio/README.rst

```diff
@@ -76,13 +76,16 @@ The Bluetooth Low Energy and IEEE 802.15.4 functionalities can operate simultane
 Sysbuild Kconfig options
 ========================
 
-To enable the firmware, use the sysbuild configuration of :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO`.
+To enable the firmware, use the sysbuild configuration ``SB_CONFIG_NETCORE_IPC_RADIO``.
```

Comment:
not sure why we aren't using the :kconfig: syntax, it works?

Reply:
@peknis informed me that Sysbuild Kconfig options are not linking properly yet.

Reply:
Haven't seen it working. It does not create a link to the Kconfig option description.

Reply:
It doesn't link but it is a valid tag. I don't think linking is possible because sysbuild uses a different Kconfig tree @gmarull thoughts? But I think we should use :kconfig: so we can do a grep for these and maybe some verification/validation/linking in future

Reply:
do these Kconfig options actually map to an existing Kconfig option? ie, if we link to the option without `SB_` prefix, would it work?

Reply:
No, they are new

Reply:
Then we have no easy option. I'd avoid using the Kconfig role, we can always grep for `SB_CONFIG_.*`.